pymc3/tests/test_examples.py

```diff
@@ -278,7 +277,7 @@ def test_run(self):
                 "theta": np.array(5, dtype="f"),
             }
             step_one = pm.Metropolis([model["theta_interval__"], model["psi_logodds__"]])
-            step_two = pm.BinaryMetropolis([model.z])
+            step_two = pm.BinaryMetropolis([model.rvs_to_values[model["z"]]])
```

Comment:
Do we need to pass the `value_var` explicitly, or it works as well with the `rv_val` (i.e., `model.z`)? 

The test passes but I didn't have the patience to check if it affected sampling.

Reply:
The value variables are given to the step methods.  See [here](https://github.com/pymc-devs/pymc3/blob/v4/pymc3/sampling.py#L198).

Reply:
I meant when specifying manually the steps as in this test

Reply:
In general, step methods require value variables as their `vars` argument/field, so it looks like your manual setup for that step method in the test is correct.

Reply:
Thanks, that was exactly my hunch.

Should we add some logic where we check if a variable manually assigned to a stepper contains a `value_var` and if so use that instead?

That would make manual assignments less cumbersome and error prone. What do you think?

Reply:
We only need to state what kind of variables they should be in a docstring.  Aside from that, we could perhaps check that the `vars` are actually value variables in the step method's `model`.